README.md

```diff
@@ -1,9 +1,502 @@
+  - sets the [force-downtime](#force-downtime) value on the [workload](#workload-layer) or [namespace](#namespace-layer) layer
+- <span id="downscaler/downscale-replicas">downscaler/downscale-replicas</span>:
+  - sets the [downscale-replicas](#downscale-replicas) value on the [workload](#workload-layer) or [namespace](#namespace-layer) layer
+- <span id="downscaler/grace-period">downscaler/grace-period</span>:
+  - sets the [grace-period](#grace-period) value on the [workload](#workload-layer) or [namespace](#namespace-layer) layer
+
+### Arguments
+
+CLI arguements set [layer values](#values) and runtime configuration at the start of the program. See the [layers concept](#layers) for more details on which of the layers [values](#values) will be used.
+
+Layer Values:
+
+- <span id="--upscale-period">--upscale-period</span>:
+  - sets the [upscale-period](#upscale-period) value on the [cli layer](#cli-layer)
+- <span id="--default-uptime">--default-uptime</span>:
+  - sets the [default-uptime](#default-uptime) value on the [cli layer](#cli-layer)
+- <span id="--downscale-period">--downscale-period</span>:
+  - sets the [downscale-period](#downscale-period) value on the [cli layer](#cli-layer)
+- <span id="--default-downtime">--default-downtime</span>:
+  - sets the [default-downtime](#default-downtime) value on the [cli layer](#cli-layer)
+- <span id="--downtime-replicas">--downtime-replicas</span>:
+  - sets the [downscale replicas](#downscale-replicas) value on the [cli layer](#cli-layer)
+- <span id="--explicit-include">--explicit-include</span>:
+  - sets the [exclude value](#exclude) on the [cli layer](#cli-layer) to true, which excludes every workload unless the exclude value on the [workload](#workload-layer) or [namespace](#namespace-layer) layer is set to false. See the [layers concept](#layers) for more details.
+
+Runtime Configuration:
+
+- <span id="--dry-run">--dry-run</span>:
+  - boolean
+  - sets the downscaler into dry run mode, which makes it just print what it would have done, instead of actually doing it
+  - default: false
+- <span id="--debug">--debug</span>:
+  - boolean
+  - makes the downscaler print debug information
+  - default: false
+- <span id="--once">--once</span>:
+  - boolean
+  - makes the downscaler exit after one scan
+  - default: false
+- <span id="--interval">--interval</span>:
+  - [duration](#duration)
+  - sets the time between scans
+  - default: 30s
+- <span id="--namespace">--namespace</span>:
+  - comma seperated list of namespaces (`some-ns,other-ns` or `some-ns, other-ns`)
+  - makes the downscaler only get the specified namespaces
+  - default: all namespaces
+- <span id="--include-resources">--include-resources</span>:
+  - comma seperated list of [scalable resources](#scalable-resources) (`deployments,statefulsets` or `deployments, statefulsets`)
+  - restricts the downscaler to only scale workloads of this resource kind
```

Comment:
This almost makes it sound as if the field is left empty it allows all kinds of downscaling. I would rephrase to something like `Enables downscaling for the specified resources` or something similar